sysl2/sysl/cmd_codegen.go

```diff
@@ -29,7 +30,7 @@ func (p *codegenCmd) Configure(app *kingpin.Application) *kingpin.CmdClause {
 
 func (p *codegenCmd) Execute(args ExecuteArgs) error {
 
-	output, err := GenerateCode(&p.CmdContextParamCodegen, args.Module, args.ModuleAppName, args.Filesystem, args.Logger)
+	output, err := GenerateCode(&p.CmdContextParamCodegen, args.Module, p.appName, args.Filesystem, args.Logger)
```

Comment:
What happens to the `ModuleAppName` field? Can it be removed?

Reply:
Opened PR by a mistake.... 😞
Yes ModuleAppName is not required. Will remove it